src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/pricing/importer/PriceImporter.kt

```diff
@@ -77,8 +72,8 @@ class PriceImporter(private val locationRepo: LocationRepository,
             try {
                 priceRepo.deleteAll()
 
-                geoameySpreadsheetProvider.get(geoPricesFile).use { prices -> import(prices, Supplier.GEOAMEY) }
-                sercoSpreadsheetProvider.get(sercoPricesFile).use { prices -> import(prices, Supplier.SERCO) }
+                sercoPrices.get(geoPricesFile).use { prices -> import(prices, Supplier.GEOAMEY) }
```

Comment:
you could write ` sercoPrices.get(geoPricesFile).use { import(it, Supplier.GEOAMEY) }`